medusa/providers/torrent/html/hebits.py

```diff
@@ -0,0 +1,199 @@
+
+            # Continue only if at least one release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return items
+
+            # Skip column headers
+            for row in torrent_rows:
+                try:
+                    heb_eng_title = row.find('div', class_='bTitle').find(href=re.compile("details\.php")).find('b').get_text(strip=True)
+                    index = heb_eng_title.index('/')
+                    title = heb_eng_title[index+1:]
+
+                    download_url = urljoin(self.url, row.find('div', class_='bTitle').find(href=re.compile("download\.php"))['href'])
+                    
+                    if not all([title, download_url]):
+                        continue
+
+                    seeders = try_int(row.find('div', class_='bUping').get_text(strip=True))
+                    leechers = try_int(row.find('div', class_='bDowning').get_text(strip=True))
+
+                    # Filter unseeded torrent
+                    if seeders < min(self.minseed, 1):
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      " minimum seeders: {0}. Seeders: {1}",
+                                      title, seeders)
+                        continue
+
+                    torrent_size = row.find('div', class_='bSize').get_text(strip=True)
+                    size = convert_size(torrent_size[5:], sep='') or -1
+
+                    item = {
+                        'title': title,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                        'pubdate': None,
+                    }
+                    if mode != 'RSS':
+                        log.debug('Found result: {0} with {1} seeders and {2} leechers',
+                                  title, seeders, leechers)
+
+                    items.append(item)
+                except (AttributeError, TypeError, KeyError, ValueError, IndexError):
+                    log.error('Failed parsing provider. Traceback: {0!r}',
+                              traceback.format_exc())
+
+        return items
+
+    def login(self):
+        """Login method used for logging in before doing search and torrent downloads."""
+        cookies = dict_from_cookiejar(self.session.cookies)
+        if any(cookies.values()) and cookies.get('uid'):
+            return True
+
+        login_params = {
+            'username': self.username,
+            'password': self.password,
+        }
+
+        response = self.session.post(self.urls['login'], data=login_params)
+
+        if not response or not response.text:
+            log.warning('Unable to connect to provider')
+            return False
+
+        if response.text == 'OK':
```

Comment:
Are you sure this is what you want? That expects the response.text to be exactly ```OK``` -  if you want to check if the response contains ```OK``` you would do it like this:
```
if 'OK' in response.text:
    return True
```
Same about the rest below.

Reply:
Yes, this is the exact response. Tested and it's working.